.github/workflows/test.yml

```diff
@@ -54,7 +54,7 @@ jobs:
     name: Matrix Test
     needs: build
     runs-on: ubuntu-latest
-    timeout-minutes: 20
+    timeout-minutes: 25
```

Comment:
Adding 5 more, have seen on this PR and others `Error: The action has timed out.` during Acceptance Tests execution